internal/schema2/property_type.go

```diff
@@ -18,6 +18,7 @@ const (
 	PTProcessList                 PropertyType = "ProcessList"
 	PTTerminateOnLastHandleClosed PropertyType = "TerminateOnLastHandleClosed"
 	PTSharedMemoryRegion          PropertyType = "SharedMemoryRegion"
+	PTContainerCredentialGuard    PropertyType = "ContainerCredentialGuard" // This was not originally apart of this struct
```

Comment:
Do we need this comment here?

Reply:
Thought it would be helpful in the future to see an addition that wasnt generated from the schema run.

Reply:
Then maybe update this to be "this field is not automatically generated with swagger currently" or something.